cmake/Modules/MacroUtilities.cmake

```
@@ -758,10 +772,15 @@ FUNCTION(TIMEMORY_INSTALL_LIBRARIES)
 
         install(
             TARGETS ${_LIB}
-            DESTINATION ${LIB_DESTINATION}
+            ${_dst}
             EXPORT ${PROJECT_NAME}-library-depends
             OPTIONAL)
 
+        if (WIN32 AND SHARED_LIBS)
```

```suggestion
        if (WIN32 AND "${_LIB}" IN_LIST SHARED_LIBS)
```
I think this is what you want. Are PDB files only generated for shared libraries?

---

Yes, pdb files are for shared libraries only.